app/src/lib/fetch-data.ts

```diff
@@ -107,6 +107,15 @@ const transformScore = (scores: any, sport: Sport) => {
   }
 }
 
+export const fetchCurrentLeaguesIds = async (sport: Sport) => {
+  const res = await fetchSportData(sport, '/leagues', new URLSearchParams())
```

Comment:
To address your concern about the number of requests caused here - maybe we can pass a parameter to revalidate the cache only once per day.

Reply:
It should actually be on the league details request, but I can't put a comment there.